mxnet/buildspec.yml

```diff
@@ -39,10 +39,10 @@ images:
   BuildMXNetCPUTrainPy3DockerImage:
     <<: *TRAINING_REPOSITORY
     build: &MXNET_CPU_TRAINING_PY3 false
-    image_size_baseline: 1786
+    image_size_baseline: 4899
```

Comment:
Are these new image baseline sizes coming from the mx 1.6 build spec? 

We should likely follow the convention we have in TF and version these buildspecs. Would you mind creating a copy of this as 1-7, and an additional file for the 1-8 MXNet so people have easy access to what the build spec should be?

Reply:
they are temp changes to run 1.7 on PR